.gitignore

```diff
@@ -165,4 +165,5 @@ cython_debug/
 # PAX-custom rules
 log.txt
 alerts.db
+*.sqlit*
```

Comment:
```suggestion
*.sqlite*
```

Reply:
This was done on purpose and is not a typo